app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  # load_and_authorize_resourcecls
+
```

Comment:
- [ ] Same here, It's considered good practice to have a readable code that is clean from any commented(unused) code. Please remove this commented code.

Reply:
Done